tokens/transfer-tokens/seahorse/programs_py/seahorse.py

```diff
@@ -0,0 +1,57 @@
+# seahorse
+# Built with Seahorse v0.2.7
+
+from seahorse.prelude import *
+
+declare_id('5KCV219sxBAZMfXWP5EZ57D6K9568krgPKGe1Lq2nkxH')
+
+@instruction
+def create_token(
+  mint: Empty[TokenMint],
+  signer: Signer
+):
+  mint.init(
+    payer = signer,
+    decimals = 6,
+    authority = signer
+  )
+
+@instruction
+def mint_token(
+  mint: TokenMint,
+  recipient: TokenAccount,
+  signer: Signer,
+  amount: u64
+):
+  mint.mint(
+    authority = signer,
+    to = recipient,
+    amount = amount * u64(10) ** 6
```

Comment:
Can we use the decimals field here instead of a hard-code?

Reply:
Should be good now